### ISSUE TITLE
Wrong import path if paths.root is defined in hardhat config

### ISSUE THREAD

**Issue by emretepedev** [frangio/hardhat-exposed#24]
I set the root path in my hardhat config.
`hardhat.config.ts`
```ts
const config: HardhatUserConfig = {
  paths: {
    root: './src',
  },
  exposed: {
    prefix: 'x_',
  },
}
```

This is my Example contract.
`src/contracts/Example.sol`
```solidity
// SPDX-License-Identifier: MIT
pragma solidity ^0.8.0;

contract Example {
    address private foo = address(this);
}
```

But it seems the wrong path when importing the base contract.
**Expected** `src/contracts-exposed/Example.sol` output:
```solidity
import "../contracts/Example.sol";

contract X_Example is Example {
   // ...
}
```

**Actual** `src/contracts-exposed/Example.sol` output:
```solidity
import "../../contracts/Example.sol"; // <--- wrong path here

contract X_Example is Example {
    // ...
}
```

**Comment by emretepedev** [comment]
Any fix here for path issue?

**Comment by nicolasegimenez** [comment]
You try to change the prefix for  "_"?

**Comment by frangio** [comment]
Fixed in 0de23e8b4c8f110042d4351c14e83c9567eefe20.
